perfkitbenchmarker/vm_util.py

```diff
@@ -77,6 +77,9 @@
 flags.DEFINE_integer('background_cpu_threads', None,
                      'Number of threads of background cpu usage while '
                      'running a benchmark')
+flags.DEFINE_integer('background_network_mbits', None,
+                     'Amount of background network traffic while '
```

Comment:
I suppose it's more important to specify network. Beyond that, having the name read nicely is a plus. If ```background_network_mbits_per_sec``` works, that would be best, but if that's too long, ```background_network_mbits_sec``` would suffice for me.